src/libraries/System.Private.CoreLib/src/System/Globalization/CultureInfo.cs

```diff
@@ -1103,7 +1103,9 @@ public static CultureInfo GetCultureInfo(string name, string altName)
             }
             catch (ArgumentException)
             {
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly, combination of arguments used
                 throw new CultureNotFoundException("name/altName", SR.Format(SR.Argument_OneOfCulturesNotSupported, name, altName));
+#pragma warning restore CA2208
```

Comment:
I think we need to fix the analyzer to only pay attention to ArgumentException, ArgumentNullException, and ArgumentOutOfRangeException.  We don't know anything about what constructors are exposed and what their parameters mean for other arbitrary exceptions derived from these.  Then this change should be reverted.

Reply:
> I think we need to fix the analyzer to only pay attention to ArgumentException, ArgumentNullException, and ArgumentOutOfRangeException. 

By my understanding the analyzer should apply for all derived types as for all derived Exceptions `message` and `parameterName` positions are different than parent ArgumentException  [CA2208](https://docs.microsoft.com/en-us/visualstudio/code-quality/ca2208)

> We don't know anything about what constructors are exposed and what their parameters mean for other arbitrary exceptions derived from these

I do not really understand why or what you mean by this, but the contractors of this CultureNotFoundException looks pretty similar to other derived types. Anyways the analyzer checks the position of `message` and `parameterName` parameters positions of the constructor with the exact exception positions, not comparing with one fixed structure

Reply:
Good point thanks, @bartonjs @terrajobst do we want to support all `ArgumentExcpetion` derived types defined in .Net Core or only some of them selectively? if so which ones?

Reply:
I think it's fair to pay attention to all ArgumentException-derived types (including itself) that have a `paramName` parameter, including user defined types.

Reply:
That makes sense, thanks @bartonjs, i will see how effectively i can apply that rule, or if the fixed type check is more efficient in the implementation. I have added that improvement into the analyzer issue https://github.com/dotnet/runtime/issues/33763#issuecomment-628279783. I think for now we can merge this PR after CI legs passes green

Reply:
Since this call -is- using a `paramName` based ctor the analyzer would still fire.

I agree that the throw doesn't have the context to know which of the two failed.